packaging_automation/tests/test_update_docker.py

```diff
@@ -21,7 +24,8 @@
 
 def setup_module():
     if not os.path.exists("docker"):
-        run("git clone https://github.com/citusdata/docker.git")
+        # TODO Remove branch name after merge of docker/PR# 281
```

Comment:
Here branch name will be removed after merging https://github.com/citusdata/docker/pull/281